charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -148,6 +148,9 @@ spec:
             - "--timeout={{ .Values.provisioner.timeout }}"
             - "--leader-election=true"
             - "--extra-create-metadata=true"
+{{- if .Values.provisioner.snapshotter.args.enableVolumeGroupSnapshots }}
+            - "--enable-volume-group-snapshots=true"
```

Comment:
Maybe set it to `false` otherwise? When the external-snapshotter changes the default, it should not suddenly get enabled when it is expected to be disabled.